eclass/autotools.eclass

```diff
@@ -65,7 +65,7 @@ inherit libtool
 # Do NOT change this variable in your ebuilds!
 # If you want to force a newer minor version, you can specify the correct
 # WANT value by using a colon:  <PV>:<WANT_AUTOMAKE>
-_LATEST_AUTOMAKE=( 1.16.1:1.16 1.15.1:1.15 )
+_LATEST_AUTOMAKE=( 1.16.3:1.16 1.16.1:1.16 )
```

Comment:
This does not make sense.
If you want that latest mean only `1.16` slot, then use:
```
_LATEST_AUTOMAKE=( 1.16.3:1.16 )
```